packages/console/src/components/SubmitFormChangesActionBar/index.module.scss

```diff
@@ -4,16 +4,39 @@
   position: sticky;
   bottom: 0;
   width: 100%;
-  height: 60px;
-  border: 1px solid var(--color-line-divider);
-  display: flex;
-  padding: _.unit(3) _.unit(8);
-  justify-content: flex-end;
-  background-color: var(--color-float);
-  box-shadow: var(--shadow-3);
-  border-radius: 12px 12px 0 0;
 
-  > :not(:first-child) {
-    margin-left: _.unit(3);
+  .actionBar {
+    height: 60px;
+    border: 1px solid var(--color-line-divider);
+    display: flex;
+    padding: _.unit(3) _.unit(8);
+    justify-content: flex-end;
+    background-color: var(--color-float);
+    box-shadow: var(--shadow-3);
+    border-radius: 12px 12px 0 0;
+
+    > :not(:first-child) {
+      margin-left: _.unit(3);
+    }
+  }
+
+  &.display {
+    height: 60px;
+
+    .actionBar {
+      transform: translateY(0);
+      transition: transform 0.3s ease-in;
+    }
+  }
+
+  &.hide {
+    height: 0;
+    overflow: hidden;
+    transition: height 0.3s ease-out 0.1s;
+
+    .actionBar {
+      transform: translateY(100%);
+      transition: transform 0.3s ease-out;
+    }
```

Comment:
These `.hide` styles can be provided by default

Reply:
Updated.